### ISSUE TITLE
Pagination not usable in admin when adding an item

### ISSUE THREAD

**Issue by GildDart** [omeka/plugin-ExhibitBuilder#131]
Hello,

When I want to add or search for an item I can't access to the pagination which passes under the first blocks.
I am using Chrome with Mac OS but it is the same with FireFox and Safari.

Omeka 3.0.1 / ExhibitBuilder 3.5

![Capture d’écran 2022-02-02 à 10 39 25](https://user-images.githubusercontent.com/24959970/152130554-2ac333fb-c9b4-4fa2-acac-b175def8580e.png)

![Capture d’écran 2022-02-02 à 10 38 54](https://user-images.githubusercontent.com/24959970/152130605-9a12be06-8998-4460-8e5a-25d8a0d20f1c.png)

Thanks.
Gil

**Comment by zerocrates** [comment]
Thanks for the report: we'll look into it.

It looks to me like this is a consequence of the longer strings in the French translation making the two buttons push the pagination underneath the results.

**Comment by GildDart** [comment]
Thanks you for your quick reply.
You are right it is the French translation which is too long. I tryed in English and it is OK.

I modified the French translation 

Show Search Form / Afficher le formulaire de recherche -> Afficher la recherche
Hide Search Form / Masquer le formulaire de recherche -> Masquer la recherche

Can I do the modifications somewhere ?

To be noted that it would be great if the window was rezisable ;-)

Gil
